server/application-data/application-data.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
we can remove this file as well as openapi.yaml,load.json files because we already added it as part of pull request 163 and there are no additional changes in this pull request

Reply:
done, removed openapi.json